docker-compose.yaml

```diff
@@ -28,13 +28,28 @@ services:
     networks:
       - default
 
+  validate:
+    depends_on:
+      postgres:
+        condition: service_healthy
+    image: openfga/openfga:latest
+    container_name: validate
+    environment:
+      - OPENFGA_DATASTORE_ENGINE=postgres
+      - OPENFGA_DATASTORE_URI=postgres://postgres:password@postgres:5432/postgres?sslmode=disable
+    command: validate-models
+    networks:
+      - default
+
   openfga:
     depends_on:
+      validate:
+        condition: service_completed_successfully
```

Comment:
what is the goal here, that if there are invalid models the service shouldn't start? i don't think we should do that. what we _could_ do that if the latest model has errors, then the service won't start.

in any case you have to change the `validate-models` command so that if there is at least one error, it throws a non-zero exit code to indicate error, because right now it doesn't work like that

<img width="1439" alt="image" src="https://github.com/openfga/openfga/assets/5374887/5e74d16c-3954-4359-a5bf-2040e71ab1c2">